packages/fast-components-react-base/README.md

```diff
@@ -1,7 +1,5 @@
 # FAST Components React Base
-A set of React components without styles.
-
-These can be combined with JSS styles to create styled components or used as-is.
+Base components are the structural, semantic, and interactive core of a piece of UI without any explicit visual representation. For the most part, base components are only as opinionated as they have to be to support the semantic and structural goals. In rare cases, there are structural changes made to support external styling (such as native elements like an input of type checkbox). Base components are useful when building out a new design system, or when you have two components that require the same base functionality and structure, but completely different visual represenations (button vs flipper).

```

Comment:
When referring to structural or semantic i'd like to include accessibility as an example.  We understand structural, but, others may not.  It's important to point out that we are opinionated as pertaining to semantic or structural goals, for example of accessibility.  People get accessibility.  It's the primary thing that made MWF successful.  Let's use it here as well.  It's already understood.

Reply:
resolved collaboratively with @chrisdholt and @janechu